docs/developer_guide.md

```diff
@@ -0,0 +1,42 @@
+# Developer Guide
+
+This repository provides a sandbox environment that can be used to develop and test Continuous Compliance workflows and integrations. 
+
+This document provides information about how to enter the sandbox, configure git and gh authentication, and provides a general overview on the default resources included in the sandbox. 
+
+## Enter the Sandbox Workspace
```

Comment:
Nit: Maybe we should change the word sandbox. I think the use of this word could be confusion because it isn't a true sandbox. It is sharing resources and mounting file systems from the host.